CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
--
+- Added the ability to hide compliance check result for helper rule, we will
+  scan for "ocp-hide-rule" in the warning, and if it exist we will not show
+  the rule in the compliance check result.
```

Comment:
hmm, not sure if this should be an enhancement or internal change, because this code change won't be visible to the end user, right?

Reply:
that's a good idea, I will do that